parser-typechecker/src/Unison/TermPrinter.hs

```diff
@@ -51,9 +51,11 @@ pretty :: Var v => PrettyPrintEnv -> Term v a -> Pretty ColorText
 pretty env = PP.syntaxToColor . pretty0 env emptyAc . printAnnotate env
 
 prettyBlock :: Var v => Bool -> PrettyPrintEnv -> Term v a -> Pretty ColorText
-prettyBlock elideUnit env =
-  PP.syntaxToColor . pretty0 env (emptyBlockAc { elideUnit = elideUnit })
-                   . printAnnotate env
+prettyBlock elideUnit env = PP.syntaxToColor . prettyBlock' elideUnit env
+
+prettyBlock' :: Var v => Bool -> PrettyPrintEnv -> Term v a -> Pretty SyntaxText
```

Comment:
Just exposing a version of this that gives back a `Pretty SyntaxText` - needed by doc rendering in codebase server.